### PR TITLE
Wip/edc 2691

### PR DESCRIPTION
**Pull request by DanielM235**

### CODE REVIEW

File: .storybook/tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+      "es2015"
+    ],
+    /* Specify library files to be included in the compilation. */
+    // "allowJs": true,                       /* Allow javascript files to be compiled. */
+    // "checkJs": true,                       /* Report errors in .js files. */
+    // "jsx": "preserve",                     /* Specify JSX code generation: 'preserve', 'react-native', or 'react'. */
+    // "declaration": true,                   /* Generates corresponding '.d.ts' file. */
+    // "declarationMap": true,                /* Generates a sourcemap for each corresponding '.d.ts' file. */
+    "declaration": false,
+    // "sourceMap": true,                     /* Generates corresponding '.map' file. */
```
Comment (greg7859) [inline .storybook/tsconfig.json]: Remove this line.
Why do you keep the commented line?

File: package.json [hunk 2]
```diff
@@ -1,12 +1,68 @@
...
+    "eslint": "^7.9.0",
+    "eslint-loader": "^4.0.2",
+    "jest": "^26.5.3",
+    "jest-puppeteer": "^4.4.0",
+    "less": "^3.12.2",
+    "less-loader": "^7.0.2",
+    "less-plugin-clean-css": "^1.5.1",
+    "mini-css-extract-plugin": "^1.0.0",
+    "puppeteer": "^2.1.1",
+    "react": "^17.0.0",
```
Comment (greg7859) [inline package.json:54]: Do you need react for the JS library ?

File: tsconfig.json [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+    "target": "es6",
+    /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019', 'ES2020', or 'ESNEXT'. */
+    "module": "es6",
+    /* Specify module code generation: 'none', 'commonjs', 'amd', 'system', 'umd', 'es2015', 'es2020', or 'ESNext'. */
+    "lib": [
+      "dom",
+      "es2015"
+    ],
+    /* Specify library files to be included in the compilation. */
+    // "allowJs": true,                       /* Allow javascript files to be compiled. */
```
Comment (greg7859) [inline tsconfig.json]: Do you need to keep the commented line (unused configuration)?

**Review by greg7859 — CHANGES_REQUESTED** [review]

File: .storybook/tsconfig.json (hunk 1 above)
Comment (DanielM235) [reply .storybook/tsconfig.json]: for easier memo access, but it does make the configuration less readable, will remove them

File: package.json (hunk 2 above)
Comment (DanielM235) [reply package.json:54]: it's for storybook, some of their extensions require it

File: .storybook/tsconfig.json (hunk 1 above)
Comment (DanielM235) [reply .storybook/tsconfig.json]: done

File: tsconfig.json (hunk 3 above)
Comment (DanielM235) [reply tsconfig.json]: done

**Review by greg7859 — APPROVED** [review]

**Pull request merged**